ui/tests/acceptance/pageobjects/accountsPage.js

```diff
@@ -0,0 +1,41 @@
+const { util } = require('util')
```

Comment:
Codacy found an issue: ['util' is assigned a value but never used.](https://app.codacy.com/gh/owncloud/ocis-accounts/pullRequest?prid=5799018)